### ISSUE TITLE
Use QC tools to catch mis-mapped data fields during import

### ISSUE THREAD

**Issue by CEJohnston** [nbnuk/nbnatlas-issues#660]
The suggestion: Mis-mapped data fields during import - caught by QC tools?
The reason: Improving data quality

Needs requirements to determine the vocabularies.

**Comment by c-van-dierkson** [comment]
Fields supplied with records can contain the wrong type of information leading to incomplete records filtering results.  Could the Atlas import be used to ID records with errors of this type?

**Comment by sophiathirza** [comment]
@c-van-dierkson this is about data quality and adding vocabularies to terms, which I have a Trello card for. I don't think that we have a problem with mis-matched fields but we do have problems with the information in the fields and it would be great if we could have time to add more vocabularies and data checking routines to the Atlas.

**Comment by c-van-dierkson** [comment]
Hi Sophie.

I searched for ‘vocab’, ‘data quality’ ‘routine’ and ‘term’ but couldn’t find the related Trello card. Can you let me know what you named it please?

Many thanks

Caroline

**Comment by sophiathirza** [comment]
@c-van-dierkson no, I can't find them in the new Trello instance. It must have been in the old one.
I have requirements for adding a vocabulary to sex, organismScope, lifeStage and vitality.

**Comment by c-van-dierkson** [comment]
Hi Sophie. Everything was copied over from the old Trello Board.

I found these... 
https://trello.com/c/ms7A3KcJ/237-controlled-vocabulary-for-sex
https://trello.com/c/LSm7NLue/243-organismscope
https://trello.com/c/4b1uEsbi/211-12-lifestage-field-to-be-made-multivalue-0
https://trello.com/c/GVmp6sv3/123-process-contents-of-vitality-field

But maybe we could have one card that combines all the controlled vocab you need when you send the requirements doc I can create that card?

**Comment by sophiathirza** [comment]
ah.. I closed some of them. I will create a card for them.
